PowershellTools.Common/Debugging/DebugEngineConstants.cs

```diff
@@ -129,5 +142,13 @@ public static class DebugEngineConstants
 
         public const string PowerShellHostProcessLogTag = "[{0}]:";
         public const string PowerShellHostProcessLogFormat = PowerShellHostProcessLogTag + "{1}";
+
+        /// <summary>
+        /// Lookup table for error messages pertaining to opening files in the various debugging scenarios.
+        /// </summary>
+        public static readonly string[] FileOpenErrorMessages = { "Failed to open file", // LOCAL

```

Comment:
string[] lookup tables are prone to error, as there's no compiletime check in regards to input and access. [Example: What if we add more DebugScenarios, but we don't immediately know we need to update this?]

Please either switch/case in a method when passing in the DebugScenario enum, or have a typed mapping.